src/components/ToDoItem/ToDoItem.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
```

Comment:
[Not critical]
It's better to name this type something like `ToDoItemProps`, so that the name describes this type the most precisely.